docs/specs/forecast-api.md

```diff
@@ -176,14 +160,14 @@ Coordinates:
   * latitude   (latitude) float32 3kB -80.0 -79.75 -79.5 ... 79.5 79.75 80.0
   * flight_level      (flight_level) int16 2B 300
   * time       (time) datetime64[ns] 8B 2024-07-01T12:00:00
+    forecast_reference_time (time) datetime64[ns] 8B 2024-07-01T06:00:00
 Data variables:
     contrails   (longitude, latitude, flight_level, time) float32 4MB ...
 Attributes:
-    forecast_reference_time:  "2024-07-01T06:00:00Z"
     aircraft_class: "default"
 ```
 
-Note that `forecast_reference_time` in the dataset level *Attributes* has the same definition as
+Note that `forecast_reference_time` non-dimension coordinate has the same definition as
 `model_run_at` in the API response header.
```

Comment:
Are we still including `model_run_at`? Is this comment necessary anymore?

Reply:
I don't think so (and agreeing w/ you suggestion below to just include `forecast_reference_time` in the geoJSON properties).

We can strike it here (and elsewhere), thereby making the `model_*` quantities something of an internal definition/concern only.